Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Overrides/PrefabOverridePublicHandler.h

```diff
@@ -19,12 +19,14 @@ namespace AzToolsFramework
         class PrefabFocusInterface;
         class PrefabSystemComponentInterface;
 
-        class PrefabOverridePublicHandler : private PrefabOverridePublicInterface
+        class PrefabOverridePublicHandler : public PrefabOverridePublicRequestBus::Handler
         {
```

Comment:
curious why we can remove `private PrefabOverridePublicInterface` and it still works?

Reply:
The handler is templated off of the Interface

Reply:
There's some info here with regards to choosing private/public EBus implementations:

https://www.o3de.org/docs/user-guide/programming/components/entity-system-pg-components-ebuses-best-practices/#making-functions-public-or-protected